polling_stations/apps/data_collection/management/commands/import_west_oxfordshire.py

```diff
@@ -3,15 +3,46 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E07000181"
-    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019WO.tsv"
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019WO.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    allow_station_point_from_postcode = False
+
+    def station_record_to_dict(self, record):
+
+        if record.polling_place_id == "6700":
+            record = record._replace(polling_place_postcode="OX18 3PU")
+            record = record._replace(polling_place_easting="430059")
+            record = record._replace(polling_place_northing="207340")
+
+        return super().station_record_to_dict(record)
 
     def address_record_to_dict(self, record):
         rec = super().address_record_to_dict(record)
         uprn = record.property_urn.strip().lstrip("0")
 
+        # if uprn == "10094864970":  # Annexe At Laurel Mount, 4 North Street, Middle Barton, Chipping Norton, Oxon
+        #     print(record)
+
+        #     record._replace(addressline6="OX7 7BJ")
+        #     print(record)
```

Comment:
there's a few bits of debug code to clean up here

Reply:
:+1: